___Execucao_com 2 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.


FFT_SERIAL
  C version

  Demonstrate an implementation of the Fast Fourier Transform
  of a complex data vector.

  Accuracy check:

    FFT ( FFT ( X(1:N) ) ) == N * X(1:N)

             N      NITS    Error

             2     10000  7.859082e-17
             4     10000  1.209837e-16
             8     10000  6.820795e-17
            16     10000  1.438671e-16
            32      1000  1.331210e-16
            64      1000  1.776545e-16
           128      1000  1.929043e-16
           256      1000  2.092319e-16
           512       100  1.927488e-16
          1024       100  2.312093e-16
          2048       100  2.445006e-16
          4096       100  2.476589e-16
          8192        10  2.571250e-16
         16384        10  2.736298e-16
         32768        10  2.924127e-16
         65536        10  2.833553e-16
        131072         1  3.142312e-16
        262144         1  3.216005e-16
        524288         1  3.282664e-16
       1048576         1  3.284479e-16
       2097152         1  3.509548e-16
       4194304         1  3.560792e-16
       8388608         1  3.709655e-16
      16777216         1  3.651822e-16
      33554432         1  3.816475e-16
      67108864         1  4.072041e-16
Execution Time: 50.036737435963005
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:50
	Energy:	0.0017 kWh
	CO2eq:	0.8219 g
	This is equivalent to:
	0.0076 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:50
	Energy:	0.0017 kWh
	CO2eq:	0.8219 g
	This is equivalent to:
	0.0076 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 2 Treads_concluida___
___Execucao_com 4 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.


FFT_SERIAL
  C version

  Demonstrate an implementation of the Fast Fourier Transform
  of a complex data vector.

  Accuracy check:

    FFT ( FFT ( X(1:N) ) ) == N * X(1:N)

             N      NITS    Error

             2     10000  7.859082e-17
             4     10000  1.209837e-16
             8     10000  6.820795e-17
            16     10000  1.438671e-16
            32      1000  1.331210e-16
            64      1000  1.776545e-16
           128      1000  1.929043e-16
           256      1000  2.092319e-16
           512       100  1.927488e-16
          1024       100  2.312093e-16
          2048       100  2.445006e-16
          4096       100  2.476589e-16
          8192        10  2.571250e-16
         16384        10  2.736298e-16
         32768        10  2.924127e-16
         65536        10  2.833553e-16
        131072         1  3.142312e-16
        262144         1  3.216005e-16
        524288         1  3.282664e-16
       1048576         1  3.284479e-16
       2097152         1  3.509548e-16
       4194304         1  3.560792e-16
       8388608         1  3.709655e-16
      16777216         1  3.651822e-16
      33554432         1  3.816475e-16
      67108864         1  4.072041e-16
Execution Time: 38.028549944981933
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:38
	Energy:	0.0014 kWh
	CO2eq:	0.6493 g
	This is equivalent to:
	0.0060 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:38
	Energy:	0.0014 kWh
	CO2eq:	0.6493 g
	This is equivalent to:
	0.0060 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 4 Treads_concluida___
___Execucao_com 6 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.


FFT_SERIAL
  C version

  Demonstrate an implementation of the Fast Fourier Transform
  of a complex data vector.

  Accuracy check:

    FFT ( FFT ( X(1:N) ) ) == N * X(1:N)

             N      NITS    Error

             2     10000  7.859082e-17
             4     10000  1.209837e-16
             8     10000  6.820795e-17
            16     10000  1.438671e-16
            32      1000  1.331210e-16
            64      1000  1.776545e-16
           128      1000  1.929043e-16
           256      1000  2.092319e-16
           512       100  1.927488e-16
          1024       100  2.312093e-16
          2048       100  2.445006e-16
          4096       100  2.476589e-16
          8192        10  2.571250e-16
         16384        10  2.736298e-16
         32768        10  2.924127e-16
         65536        10  2.833553e-16
        131072         1  3.142312e-16
        262144         1  3.216005e-16
        524288         1  3.282664e-16
       1048576         1  3.284479e-16
       2097152         1  3.509548e-16
       4194304         1  3.560792e-16
       8388608         1  3.709655e-16
      16777216         1  3.651822e-16
      33554432         1  3.816475e-16
      67108864         1  4.072041e-16
Execution Time: 34.868039029999636
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:35
	Energy:	0.0013 kWh
	CO2eq:	0.6292 g
	This is equivalent to:
	0.0059 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:35
	Energy:	0.0013 kWh
	CO2eq:	0.6292 g
	This is equivalent to:
	0.0059 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 6 Treads_concluida___
___Execucao_com 8 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.


FFT_SERIAL
  C version

  Demonstrate an implementation of the Fast Fourier Transform
  of a complex data vector.

  Accuracy check:

    FFT ( FFT ( X(1:N) ) ) == N * X(1:N)

             N      NITS    Error

             2     10000  7.859082e-17
             4     10000  1.209837e-16
             8     10000  6.820795e-17
            16     10000  1.438671e-16
            32      1000  1.331210e-16
            64      1000  1.776545e-16
           128      1000  1.929043e-16
           256      1000  2.092319e-16
           512       100  1.927488e-16
          1024       100  2.312093e-16
          2048       100  2.445006e-16
          4096       100  2.476589e-16
          8192        10  2.571250e-16
         16384        10  2.736298e-16
         32768        10  2.924127e-16
         65536        10  2.833553e-16
        131072         1  3.142312e-16
        262144         1  3.216005e-16
        524288         1  3.282664e-16
       1048576         1  3.284479e-16
       2097152         1  3.509548e-16
       4194304         1  3.560792e-16
       8388608         1  3.709655e-16
      16777216         1  3.651822e-16
      33554432         1  3.816475e-16
      67108864         1  4.072041e-16
Execution Time: 34.144899976905435
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:34
	Energy:	0.0013 kWh
	CO2eq:	0.6058 g
	This is equivalent to:
	0.0056 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:34
	Energy:	0.0013 kWh
	CO2eq:	0.6058 g
	This is equivalent to:
	0.0056 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 8 Treads_concluida___
___Execucao_com 10 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.


FFT_SERIAL
  C version

  Demonstrate an implementation of the Fast Fourier Transform
  of a complex data vector.

  Accuracy check:

    FFT ( FFT ( X(1:N) ) ) == N * X(1:N)

             N      NITS    Error

             2     10000  7.859082e-17
             4     10000  1.209837e-16
             8     10000  6.820795e-17
            16     10000  1.438671e-16
            32      1000  1.331210e-16
            64      1000  1.776545e-16
           128      1000  1.929043e-16
           256      1000  2.092319e-16
           512       100  1.927488e-16
          1024       100  2.312093e-16
          2048       100  2.445006e-16
          4096       100  2.476589e-16
          8192        10  2.571250e-16
         16384        10  2.736298e-16
         32768        10  2.924127e-16
         65536        10  2.833553e-16
        131072         1  3.142312e-16
        262144         1  3.216005e-16
        524288         1  3.282664e-16
       1048576         1  3.284479e-16
       2097152         1  3.509548e-16
       4194304         1  3.560792e-16
       8388608         1  3.709655e-16
      16777216         1  3.651822e-16
      33554432         1  3.816475e-16
      67108864         1  4.072041e-16
Execution Time: 33.418376486981288
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:33
	Energy:	0.0012 kWh
	CO2eq:	0.5510 g
	This is equivalent to:
	0.0051 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:33
	Energy:	0.0012 kWh
	CO2eq:	0.5510 g
	This is equivalent to:
	0.0051 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 10 Treads_concluida___
___Execucao_com 12 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.


FFT_SERIAL
  C version

  Demonstrate an implementation of the Fast Fourier Transform
  of a complex data vector.

  Accuracy check:

    FFT ( FFT ( X(1:N) ) ) == N * X(1:N)

             N      NITS    Error

             2     10000  7.859082e-17
             4     10000  1.209837e-16
             8     10000  6.820795e-17
            16     10000  1.438671e-16
            32      1000  1.331210e-16
            64      1000  1.776545e-16
           128      1000  1.929043e-16
           256      1000  2.092319e-16
           512       100  1.927488e-16
          1024       100  2.312093e-16
          2048       100  2.445006e-16
          4096       100  2.476589e-16
          8192        10  2.571250e-16
         16384        10  2.736298e-16
         32768        10  2.924127e-16
         65536        10  2.833553e-16
        131072         1  3.142312e-16
        262144         1  3.216005e-16
        524288         1  3.282664e-16
       1048576         1  3.284479e-16
       2097152         1  3.509548e-16
       4194304         1  3.560792e-16
       8388608         1  3.709655e-16
      16777216         1  3.651822e-16
      33554432         1  3.816475e-16
      67108864         1  4.072041e-16
Execution Time: 33.266354889026843
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:33
	Energy:	0.0012 kWh
	CO2eq:	0.5582 g
	This is equivalent to:
	0.0052 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:33
	Energy:	0.0012 kWh
	CO2eq:	0.5582 g
	This is equivalent to:
	0.0052 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 12 Treads_concluida___
___Execucao_com 14 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.


FFT_SERIAL
  C version

  Demonstrate an implementation of the Fast Fourier Transform
  of a complex data vector.

  Accuracy check:

    FFT ( FFT ( X(1:N) ) ) == N * X(1:N)

             N      NITS    Error

             2     10000  7.859082e-17
             4     10000  1.209837e-16
             8     10000  6.820795e-17
            16     10000  1.438671e-16
            32      1000  1.331210e-16
            64      1000  1.776545e-16
           128      1000  1.929043e-16
           256      1000  2.092319e-16
           512       100  1.927488e-16
          1024       100  2.312093e-16
          2048       100  2.445006e-16
          4096       100  2.476589e-16
          8192        10  2.571250e-16
         16384        10  2.736298e-16
         32768        10  2.924127e-16
         65536        10  2.833553e-16
        131072         1  3.142312e-16
        262144         1  3.216005e-16
        524288         1  3.282664e-16
       1048576         1  3.284479e-16
       2097152         1  3.509548e-16
       4194304         1  3.560792e-16
       8388608         1  3.709655e-16
      16777216         1  3.651822e-16
      33554432         1  3.816475e-16
      67108864         1  4.072041e-16
Execution Time: 33.049571788054891
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:33
	Energy:	0.0012 kWh
	CO2eq:	0.5594 g
	This is equivalent to:
	0.0052 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:33
	Energy:	0.0012 kWh
	CO2eq:	0.5594 g
	This is equivalent to:
	0.0052 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 14 Treads_concluida___
___Execucao_com 16 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.


FFT_SERIAL
  C version

  Demonstrate an implementation of the Fast Fourier Transform
  of a complex data vector.

  Accuracy check:

    FFT ( FFT ( X(1:N) ) ) == N * X(1:N)

             N      NITS    Error

             2     10000  7.859082e-17
             4     10000  1.209837e-16
             8     10000  6.820795e-17
            16     10000  1.438671e-16
            32      1000  1.331210e-16
            64      1000  1.776545e-16
           128      1000  1.929043e-16
           256      1000  2.092319e-16
           512       100  1.927488e-16
          1024       100  2.312093e-16
          2048       100  2.445006e-16
          4096       100  2.476589e-16
          8192        10  2.571250e-16
         16384        10  2.736298e-16
         32768        10  2.924127e-16
         65536        10  2.833553e-16
        131072         1  3.142312e-16
        262144         1  3.216005e-16
        524288         1  3.282664e-16
       1048576         1  3.284479e-16
       2097152         1  3.509548e-16
       4194304         1  3.560792e-16
       8388608         1  3.709655e-16
      16777216         1  3.651822e-16
      33554432         1  3.816475e-16
      67108864         1  4.072041e-16
Execution Time: 33.657946928986348
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:34
	Energy:	0.0012 kWh
	CO2eq:	0.5856 g
	This is equivalent to:
	0.0054 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:34
	Energy:	0.0012 kWh
	CO2eq:	0.5856 g
	This is equivalent to:
	0.0054 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 16 Treads_concluida___
___Execucao_com 18 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.


FFT_SERIAL
  C version

  Demonstrate an implementation of the Fast Fourier Transform
  of a complex data vector.

  Accuracy check:

    FFT ( FFT ( X(1:N) ) ) == N * X(1:N)

             N      NITS    Error

             2     10000  7.859082e-17
             4     10000  1.209837e-16
             8     10000  6.820795e-17
            16     10000  1.438671e-16
            32      1000  1.331210e-16
            64      1000  1.776545e-16
           128      1000  1.929043e-16
           256      1000  2.092319e-16
           512       100  1.927488e-16
          1024       100  2.312093e-16
          2048       100  2.445006e-16
          4096       100  2.476589e-16
          8192        10  2.571250e-16
         16384        10  2.736298e-16
         32768        10  2.924127e-16
         65536        10  2.833553e-16
        131072         1  3.142312e-16
        262144         1  3.216005e-16
        524288         1  3.282664e-16
       1048576         1  3.284479e-16
       2097152         1  3.509548e-16
       4194304         1  3.560792e-16
       8388608         1  3.709655e-16
      16777216         1  3.651822e-16
      33554432         1  3.816475e-16
      67108864         1  4.072041e-16
Execution Time: 33.731151615036651
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:34
	Energy:	0.0012 kWh
	CO2eq:	0.5930 g
	This is equivalent to:
	0.0055 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:34
	Energy:	0.0012 kWh
	CO2eq:	0.5930 g
	This is equivalent to:
	0.0055 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 18 Treads_concluida___
___Execucao_com 20 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.


FFT_SERIAL
  C version

  Demonstrate an implementation of the Fast Fourier Transform
  of a complex data vector.

  Accuracy check:

    FFT ( FFT ( X(1:N) ) ) == N * X(1:N)

             N      NITS    Error

             2     10000  7.859082e-17
             4     10000  1.209837e-16
             8     10000  6.820795e-17
            16     10000  1.438671e-16
            32      1000  1.331210e-16
            64      1000  1.776545e-16
           128      1000  1.929043e-16
           256      1000  2.092319e-16
           512       100  1.927488e-16
          1024       100  2.312093e-16
          2048       100  2.445006e-16
          4096       100  2.476589e-16
          8192        10  2.571250e-16
         16384        10  2.736298e-16
         32768        10  2.924127e-16
         65536        10  2.833553e-16
        131072         1  3.142312e-16
        262144         1  3.216005e-16
        524288         1  3.282664e-16
       1048576         1  3.284479e-16
       2097152         1  3.509548e-16
       4194304         1  3.560792e-16
       8388608         1  3.709655e-16
      16777216         1  3.651822e-16
      33554432         1  3.816475e-16
      67108864         1  4.072041e-16
Execution Time: 33.293175573926419
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:33
	Energy:	0.0013 kWh
	CO2eq:	0.5970 g
	This is equivalent to:
	0.0056 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:33
	Energy:	0.0013 kWh
	CO2eq:	0.5970 g
	This is equivalent to:
	0.0056 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 20 Treads_concluida___
___Execucao_com 22 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.


FFT_SERIAL
  C version

  Demonstrate an implementation of the Fast Fourier Transform
  of a complex data vector.

  Accuracy check:

    FFT ( FFT ( X(1:N) ) ) == N * X(1:N)

             N      NITS    Error

             2     10000  7.859082e-17
             4     10000  1.209837e-16
             8     10000  6.820795e-17
            16     10000  1.438671e-16
            32      1000  1.331210e-16
            64      1000  1.776545e-16
           128      1000  1.929043e-16
           256      1000  2.092319e-16
           512       100  1.927488e-16
          1024       100  2.312093e-16
          2048       100  2.445006e-16
          4096       100  2.476589e-16
          8192        10  2.571250e-16
         16384        10  2.736298e-16
         32768        10  2.924127e-16
         65536        10  2.833553e-16
        131072         1  3.142312e-16
        262144         1  3.216005e-16
        524288         1  3.282664e-16
       1048576         1  3.284479e-16
       2097152         1  3.509548e-16
       4194304         1  3.560792e-16
       8388608         1  3.709655e-16
      16777216         1  3.651822e-16
      33554432         1  3.816475e-16
      67108864         1  4.072041e-16
Execution Time: 34.350618289085105
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:34
	Energy:	0.0015 kWh
	CO2eq:	0.7033 g
	This is equivalent to:
	0.0065 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:34
	Energy:	0.0015 kWh
	CO2eq:	0.7033 g
	This is equivalent to:
	0.0065 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 22 Treads_concluida___
___Execucao_com 24 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.


FFT_SERIAL
  C version

  Demonstrate an implementation of the Fast Fourier Transform
  of a complex data vector.

  Accuracy check:

    FFT ( FFT ( X(1:N) ) ) == N * X(1:N)

             N      NITS    Error

             2     10000  7.859082e-17
             4     10000  1.209837e-16
             8     10000  6.820795e-17
            16     10000  1.438671e-16
            32      1000  1.331210e-16
            64      1000  1.776545e-16
           128      1000  1.929043e-16
           256      1000  2.092319e-16
           512       100  1.927488e-16
          1024       100  2.312093e-16
          2048       100  2.445006e-16
          4096       100  2.476589e-16
          8192        10  2.571250e-16
         16384        10  2.736298e-16
         32768        10  2.924127e-16
         65536        10  2.833553e-16
        131072         1  3.142312e-16
        262144         1  3.216005e-16
        524288         1  3.282664e-16
       1048576         1  3.284479e-16
       2097152         1  3.509548e-16
       4194304         1  3.560792e-16
       8388608         1  3.709655e-16
      16777216         1  3.651822e-16
      33554432         1  3.816475e-16
      67108864         1  4.072041e-16
Execution Time: 34.362639247090556
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:34
	Energy:	0.0015 kWh
	CO2eq:	0.7058 g
	This is equivalent to:
	0.0066 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:34
	Energy:	0.0015 kWh
	CO2eq:	0.7058 g
	This is equivalent to:
	0.0066 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 24 Treads_concluida___
___Execucao_com 26 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.


FFT_SERIAL
  C version

  Demonstrate an implementation of the Fast Fourier Transform
  of a complex data vector.

  Accuracy check:

    FFT ( FFT ( X(1:N) ) ) == N * X(1:N)

             N      NITS    Error

             2     10000  7.859082e-17
             4     10000  1.209837e-16
             8     10000  6.820795e-17
            16     10000  1.438671e-16
            32      1000  1.331210e-16
            64      1000  1.776545e-16
           128      1000  1.929043e-16
           256      1000  2.092319e-16
           512       100  1.927488e-16
          1024       100  2.312093e-16
          2048       100  2.445006e-16
          4096       100  2.476589e-16
          8192        10  2.571250e-16
         16384        10  2.736298e-16
         32768        10  2.924127e-16
         65536        10  2.833553e-16
        131072         1  3.142312e-16
        262144         1  3.216005e-16
        524288         1  3.282664e-16
       1048576         1  3.284479e-16
       2097152         1  3.509548e-16
       4194304         1  3.560792e-16
       8388608         1  3.709655e-16
      16777216         1  3.651822e-16
      33554432         1  3.816475e-16
      67108864         1  4.072041e-16
Execution Time: 33.960771184065379
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:34
	Energy:	0.0013 kWh
	CO2eq:	0.6172 g
	This is equivalent to:
	0.0057 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:34
	Energy:	0.0013 kWh
	CO2eq:	0.6323 g
	This is equivalent to:
	0.0059 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 26 Treads_concluida___
___Execucao_com 28 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.


FFT_SERIAL
  C version

  Demonstrate an implementation of the Fast Fourier Transform
  of a complex data vector.

  Accuracy check:

    FFT ( FFT ( X(1:N) ) ) == N * X(1:N)

             N      NITS    Error

             2     10000  7.859082e-17
             4     10000  1.209837e-16
             8     10000  6.820795e-17
            16     10000  1.438671e-16
            32      1000  1.331210e-16
            64      1000  1.776545e-16
           128      1000  1.929043e-16
           256      1000  2.092319e-16
           512       100  1.927488e-16
          1024       100  2.312093e-16
          2048       100  2.445006e-16
          4096       100  2.476589e-16
          8192        10  2.571250e-16
         16384        10  2.736298e-16
         32768        10  2.924127e-16
         65536        10  2.833553e-16
        131072         1  3.142312e-16
        262144         1  3.216005e-16
        524288         1  3.282664e-16
       1048576         1  3.284479e-16
       2097152         1  3.509548e-16
       4194304         1  3.560792e-16
       8388608         1  3.709655e-16
      16777216         1  3.651822e-16
      33554432         1  3.816475e-16
      67108864         1  4.072041e-16
Execution Time: 33.694309104932472
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:34
	Energy:	0.0013 kWh
	CO2eq:	0.6032 g
	This is equivalent to:
	0.0056 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:34
	Energy:	0.0013 kWh
	CO2eq:	0.6032 g
	This is equivalent to:
	0.0056 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 28 Treads_concluida___
___Execucao_com 30 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.


FFT_SERIAL
  C version

  Demonstrate an implementation of the Fast Fourier Transform
  of a complex data vector.

  Accuracy check:

    FFT ( FFT ( X(1:N) ) ) == N * X(1:N)

             N      NITS    Error

             2     10000  7.859082e-17
             4     10000  1.209837e-16
             8     10000  6.820795e-17
            16     10000  1.438671e-16
            32      1000  1.331210e-16
            64      1000  1.776545e-16
           128      1000  1.929043e-16
           256      1000  2.092319e-16
           512       100  1.927488e-16
          1024       100  2.312093e-16
          2048       100  2.445006e-16
          4096       100  2.476589e-16
          8192        10  2.571250e-16
         16384        10  2.736298e-16
         32768        10  2.924127e-16
         65536        10  2.833553e-16
        131072         1  3.142312e-16
        262144         1  3.216005e-16
        524288         1  3.282664e-16
       1048576         1  3.284479e-16
       2097152         1  3.509548e-16
       4194304         1  3.560792e-16
       8388608         1  3.709655e-16
      16777216         1  3.651822e-16
      33554432         1  3.816475e-16
      67108864         1  4.072041e-16
Execution Time: 34.759092055959627
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:35
	Energy:	0.0016 kWh
	CO2eq:	0.7482 g
	This is equivalent to:
	0.0070 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:35
	Energy:	0.0016 kWh
	CO2eq:	0.7482 g
	This is equivalent to:
	0.0070 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 30 Treads_concluida___
___Execucao_com 32 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.


FFT_SERIAL
  C version

  Demonstrate an implementation of the Fast Fourier Transform
  of a complex data vector.

  Accuracy check:

    FFT ( FFT ( X(1:N) ) ) == N * X(1:N)

             N      NITS    Error

             2     10000  7.859082e-17
             4     10000  1.209837e-16
             8     10000  6.820795e-17
            16     10000  1.438671e-16
            32      1000  1.331210e-16
            64      1000  1.776545e-16
           128      1000  1.929043e-16
           256      1000  2.092319e-16
           512       100  1.927488e-16
          1024       100  2.312093e-16
          2048       100  2.445006e-16
          4096       100  2.476589e-16
          8192        10  2.571250e-16
         16384        10  2.736298e-16
         32768        10  2.924127e-16
         65536        10  2.833553e-16
        131072         1  3.142312e-16
        262144         1  3.216005e-16
        524288         1  3.282664e-16
       1048576         1  3.284479e-16
       2097152         1  3.509548e-16
       4194304         1  3.560792e-16
       8388608         1  3.709655e-16
      16777216         1  3.651822e-16
      33554432         1  3.816475e-16
      67108864         1  4.072041e-16
Execution Time: 34.286806071992032
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:34
	Energy:	0.0015 kWh
	CO2eq:	0.7266 g
	This is equivalent to:
	0.0068 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:34
	Energy:	0.0015 kWh
	CO2eq:	0.7266 g
	This is equivalent to:
	0.0068 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 32 Treads_concluida___
___Execucao_com 34 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.


FFT_SERIAL
  C version

  Demonstrate an implementation of the Fast Fourier Transform
  of a complex data vector.

  Accuracy check:

    FFT ( FFT ( X(1:N) ) ) == N * X(1:N)

             N      NITS    Error

             2     10000  7.859082e-17
             4     10000  1.209837e-16
             8     10000  6.820795e-17
            16     10000  1.438671e-16
            32      1000  1.331210e-16
            64      1000  1.776545e-16
           128      1000  1.929043e-16
           256      1000  2.092319e-16
           512       100  1.927488e-16
          1024       100  2.312093e-16
          2048       100  2.445006e-16
          4096       100  2.476589e-16
          8192        10  2.571250e-16
         16384        10  2.736298e-16
         32768        10  2.924127e-16
         65536        10  2.833553e-16
        131072         1  3.142312e-16
        262144         1  3.216005e-16
        524288         1  3.282664e-16
       1048576         1  3.284479e-16
       2097152         1  3.509548e-16
       4194304         1  3.560792e-16
       8388608         1  3.709655e-16
      16777216         1  3.651822e-16
      33554432         1  3.816475e-16
      67108864         1  4.072041e-16
Execution Time: 35.285334802931175
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:35
	Energy:	0.0015 kWh
	CO2eq:	0.7227 g
	This is equivalent to:
	0.0067 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:35
	Energy:	0.0015 kWh
	CO2eq:	0.7227 g
	This is equivalent to:
	0.0067 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 34 Treads_concluida___
___Execucao_com 36 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.


FFT_SERIAL
  C version

  Demonstrate an implementation of the Fast Fourier Transform
  of a complex data vector.

  Accuracy check:

    FFT ( FFT ( X(1:N) ) ) == N * X(1:N)

             N      NITS    Error

             2     10000  7.859082e-17
             4     10000  1.209837e-16
             8     10000  6.820795e-17
            16     10000  1.438671e-16
            32      1000  1.331210e-16
            64      1000  1.776545e-16
           128      1000  1.929043e-16
           256      1000  2.092319e-16
           512       100  1.927488e-16
          1024       100  2.312093e-16
          2048       100  2.445006e-16
          4096       100  2.476589e-16
          8192        10  2.571250e-16
         16384        10  2.736298e-16
         32768        10  2.924127e-16
         65536        10  2.833553e-16
        131072         1  3.142312e-16
        262144         1  3.216005e-16
        524288         1  3.282664e-16
       1048576         1  3.284479e-16
       2097152         1  3.509548e-16
       4194304         1  3.560792e-16
       8388608         1  3.709655e-16
      16777216         1  3.651822e-16
      33554432         1  3.816475e-16
      67108864         1  4.072041e-16
Execution Time: 35.621124471072108
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:36
	Energy:	0.0016 kWh
	CO2eq:	0.7376 g
	This is equivalent to:
	0.0069 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:36
	Energy:	0.0016 kWh
	CO2eq:	0.7376 g
	This is equivalent to:
	0.0069 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 36 Treads_concluida___
___Execucao_com 38 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.


FFT_SERIAL
  C version

  Demonstrate an implementation of the Fast Fourier Transform
  of a complex data vector.

  Accuracy check:

    FFT ( FFT ( X(1:N) ) ) == N * X(1:N)

             N      NITS    Error

             2     10000  7.859082e-17
             4     10000  1.209837e-16
             8     10000  6.820795e-17
            16     10000  1.438671e-16
            32      1000  1.331210e-16
            64      1000  1.776545e-16
           128      1000  1.929043e-16
           256      1000  2.092319e-16
           512       100  1.927488e-16
          1024       100  2.312093e-16
          2048       100  2.445006e-16
          4096       100  2.476589e-16
          8192        10  2.571250e-16
         16384        10  2.736298e-16
         32768        10  2.924127e-16
         65536        10  2.833553e-16
        131072         1  3.142312e-16
        262144         1  3.216005e-16
        524288         1  3.282664e-16
       1048576         1  3.284479e-16
       2097152         1  3.509548e-16
       4194304         1  3.560792e-16
       8388608         1  3.709655e-16
      16777216         1  3.651822e-16
      33554432         1  3.816475e-16
      67108864         1  4.072041e-16
Execution Time: 34.913787552039139
CarbonTracker: WARNING - Failed to retrieve carbon intensity: Defaulting to average carbon intensity 475 gCO2/kWh.
CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: Porto Alegre, Rio Grande do Sul, BR.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:35
	Energy:	0.0016 kWh
	CO2eq:	0.7687 g
	This is equivalent to:
	0.0072 km travelled by car
CarbonTracker: WARNING - Failed to retrieve carbon intensity: Defaulting to average carbon intensity 475 gCO2/kWh.
CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: Porto Alegre, Rio Grande do Sul, BR.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:35
	Energy:	0.0016 kWh
	CO2eq:	0.7687 g
	This is equivalent to:
	0.0072 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 38 Treads_concluida___
___Execucao_com 40 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: WARNING - Failed to retrieve carbon intensity: Defaulting to average carbon intensity 141.77426 gCO2/kWh.

FFT_SERIAL
  C version

  Demonstrate an implementation of the Fast Fourier Transform
  of a complex data vector.

  Accuracy check:

    FFT ( FFT ( X(1:N) ) ) == N * X(1:N)

             N      NITS    Error

             2     10000  7.859082e-17
             4     10000  1.209837e-16
             8     10000  6.820795e-17
            16     10000  1.438671e-16
            32      1000  1.331210e-16
            64      1000  1.776545e-16
           128      1000  1.929043e-16
           256      1000  2.092319e-16
           512       100  1.927488e-16
          1024       100  2.312093e-16
          2048       100  2.445006e-16
          4096       100  2.476589e-16
          8192        10  2.571250e-16
         16384        10  2.736298e-16
         32768        10  2.924127e-16
         65536        10  2.833553e-16
        131072         1  3.142312e-16
        262144         1  3.216005e-16
        524288         1  3.282664e-16
       1048576         1  3.284479e-16
       2097152         1  3.509548e-16
       4194304         1  3.560792e-16
       8388608         1  3.709655e-16
      16777216         1  3.651822e-16
      33554432         1  3.816475e-16
      67108864         1  4.072041e-16
Execution Time: 34.681176815996878
CarbonTracker: WARNING - Failed to retrieve carbon intensity: Defaulting to average carbon intensity 141.77426 gCO2/kWh.
CarbonTracker: Average carbon intensity during training was 141.77 gCO2/kWh at detected location: Porto Alegre, Rio Grande do Sul, BR.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:35
	Energy:	0.0017 kWh
	CO2eq:	0.2399 g
	This is equivalent to:
	0.0022 km travelled by car
CarbonTracker: WARNING - Failed to retrieve carbon intensity: Defaulting to average carbon intensity 141.77426 gCO2/kWh.
CarbonTracker: Live carbon intensity could not be fetched at detected location: Porto Alegre, Rio Grande do Sul, BR. Defaulted to average carbon intensity for BR in 2021 of 141.77 gCO2/kWh. at detected location: Porto Alegre, Rio Grande do Sul, BR.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:35
	Energy:	0.0017 kWh
	CO2eq:	0.2399 g
	This is equivalent to:
	0.0022 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 40 Treads_concluida___
Alerta: Menos de 10 log_bruto
